src/material-experimental/mdc-chips/chip-input.ts

```diff
@@ -73,6 +74,9 @@ export class MatChipInput implements MatChipTextControl, AfterContentInit, OnCha
   /** Used to prevent focus moving to chips while user is holding backspace */
   private _focusLastChipOnBackspace: boolean;
 
+  /** Value for ariaDescribedby property */
+  private _ariaDescribedby?: string;
```

Comment:
Currently this attribute is causing the linter to fail since it's not referenced by any of the TS code.  Is there a way to suppress the linter for this attribute or is there another way we can handle that?

Reply:
Since it's used in a host binding, it should be changed to a public property. That'll silence the linter as well.

Reply:
Makes sense.  Changing.